src/PublishPort.jsx

```diff
@@ -27,9 +66,16 @@
                 }>
                 <TextInput id={id + "-ip-address"}
                         value={item.IP || ''}
-                        onChange={(_event, value) => onChange(idx, 'IP', value)} />
+                        validated={validationFailed?.IP ? "error" : "default"}
+                        onChange={(_event, value) => {
+                            utils.validationClear(validationFailed, "IP", onValidationChange);
+                            utils.validationDebounce(() => onValidationChange(validatePublishPort(value, "IP", validationFailed)));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1354-20231016-121055-6688b0e7-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)